subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/http/HttpAuthenticationDependencyResolutionIntegrationTest.groovy

```diff
@@ -162,6 +164,48 @@ task listJars {
         'basic and digest' | 'authentication { basic(BasicAuthentication)\ndigest(DigestAuthentication) }' | DIGEST            | ['Basic', 'Digest']
     }
 
+    @Unroll
+    @Issue("gradle/gradle#5571")
+    public void "can resolve dependencies using #authSchemeName scheme from #authScheme authenticated with http header HTTP maven repository"() {
+        given:
+        def moduleA = mavenHttpRepo.module('group', 'projectA', '1.2').publish()
+        and:
+        buildFile << """
+repositories {
+    maven {
+        url "${mavenHttpRepo.uri}"
+        credentials(HttpHeaderCredentials) { header 'Name: Value' }
```

Comment:
Changing this line to `credentials(HttpHeaderCredentials) { header 'Authorization: Header' }` makes the test pass. However, you should probably extend the fixture to record the full HTTP header.